### ISSUE TITLE
[PROJECT] Add GDPR consents and notices 

### ISSUE THREAD

**Issue by BenJam** [opencollective/opencollective#4537]
## TLDR
We are unsure that our current communications and data sharing preferences meet the standards needed. In order to ensure that we have the necessary consents we should add explicit consents and notices at appropriate points.

## Guidance

Using the Information Comissioners Office (a UK body who created this guidance while a member of the EU, following EU legislation) https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/ the most pertinent to this issue are:

- [Right to be informed](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/individual-rights/right-to-be-informed/)
- [Right to object](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/individual-rights/right-to-object/)

> We provide individuals with privacy information at the time we collect their personal data from them.

> Individuals have the absolute right to object to the processing of their personal data if it is for direct marketing purposes. 

### Communications preferences
When users sign up to Open Collective they explicitly indicate that they would be happy to receive our monthly platform newsletter. 

<img width="538" alt="Screenshot 2021-08-09 at 13 46 36" src="https://user-images.githubusercontent.com/158833/128708897-90e04e28-e3c9-4100-b12f-279ffc6bfdc3.png">

### Contribution flow

When users contribute to a project administrators can:

- use the update feature to email them
- access their public profile information (display name, twitter, github, website) and their private email address in the CSV download available in the collective's export settings. 

Users can contribute using an incognito profile, when we do this we do not share their information with collective administrators and their email addresses will not be included in update mailouts. 

### Privacy policy

Our privacy policy (linked in signup) was last updated Jan 2019 and includes the following:

> With hosts and organizers of a collective, if you are a contributor, or request reimbursement for expenses made in support, of that collective
> We will share your name and image with GitHub if you participate in a collective that is also working on a software project through GitHub.
> With vendors, consultants and other service providers who need access to such information to carry out work on our behalf.

## Norms 

What are others in this space doing?

### [Patreon](https://www.patreon.com/)

Has a full-on piecemeal notifications system in which you get to decide, for each creator you support, what you want to hear about. their [privacy policy](https://privacy.patreon.com/policies?name=patreon-privacy-policy#information-we-share-with-creators) is quite specific:

>By becoming a patron of a creator, you agree to have the following information shared with that creator:
your name and email address, and other profile information you’ve provided
any messages you send creators through Patreon
your physical address, city, state, and country
your phone number, when you have signed up to support a creator who engages with you via text messages, when you are receiving a benefit that requires shipping, or when you have signed up for event notifications via text
all information about your pledge, including amount and start date, but not your full payment card information
some aggregated and anonymized data about how you use Patreon that cannot be linked back to you or to any individual user
Creators agree to the terms of our Data Processing Agreement (DPA) when they start their membership program

### [Kickstarter](https://www.kickstarter.com/)

Kickstarter also has piecemeal notification management, but they display it a little differently. Their [privacy policy](https://www.kickstarter.com/privacy#information-thats-shared-with-creators) states:

>If a project you have backed is successfully funded, the creator will receive the email address associated with your Kickstarter account. They may also send you a survey requesting information needed to provide your reward. (For instance, they may need your mailing address, or T-shirt size.) Any information you provide in such surveys will be received by the creator.

They're very particular about the types of information that creators can ask about and about the fact they do not share private information until the point that a project is backed. 

### [GitHub Sponsors](https://github.com/sponsors)

[Sponsors'](https://docs.github.com/en/github/site-policy/github-sponsors-additional-terms) terms does not include anything specific about data sharing, instead they're relying on the general data sharing agreements in their overall policy.

## Proposed Solution

- [ ] Improve the notice provided to contributors about the information shared with collectives
- [ ] Check that incognito contributors do not have any identifying information listed in Collective contributor exports. 
- [ ] Check that incognito contributors do not receive Updates
- [ ] Add an explicit process for objecting to our data sharing agreements and requesting data removal to our ToS/privacy policy

**Comment by SudharakaP** [comment]
Maybe this is a bigger issue, but I feel as part of this we should also indicate (or give DIY options) on how they could opt out of these notifications. 😉

**Comment by BenJam** [comment]
@SudharakaP fair, we should probbably add a newsletter setting to the user's info settings.

**Comment by Betree** [comment]
Related: https://github.com/opencollective/opencollective/issues/3357

**Comment by BenJam** [comment]
issues here: what happens when we have the consents, how are we gaining retroactive consent? what's the process going to be for that?

Perosnal note: what is the process for posting project updates, are contributors automatically notified about that?

**Comment by BenJam** [comment]
Quick follow up after sync call with @znarf and @Betree: amended original proposal to remove explicit opt outs/ins and instead improve noticed and add opt-out process to ToS/Privacy Policy. 

Related: we noticed that there is an issue with information shared for organisations. Currently, contributor exports do not include contact addresses when a contributor is an organisation. We thought it would be off to include the contact email addresses for all admins associated with an org so instead we should add a contact email address for orgs. 

Also related: we may want to improve the Delete Account process (which is not accessible for user who have transactions in the ledger) as this could be used as part of an opt out process. At a high level this process would make all contributions in the ledger as incognito, remove the users profile from the site but retain all the private information that we need in order to comply with accounting, tax and legal processes.

**Comment by BenJam** [comment]
> Improve the notice provided to contributors about the information shared with collectives

@opencollective/design wondering about two approaches here:

| callout in main flow | callout in FAQ | 
| --- | --- |
| ![Frame 1](https://user-images.githubusercontent.com/158833/133818519-9a32993a-bd4b-4e5f-bbd7-d9421ba194df.png) | ![Frame 2](https://user-images.githubusercontent.com/158833/133818544-47d5bba2-b531-4873-b697-5e52da48ef32.png) |

we dont want to discourage folks from contributing under their own name but we do need to do a better job of informing them about what information we share with Collectives. What do you think?

**Comment by aminakazi** [comment]
We've concluded that the best way going forward for the GDPR notice would be these two proposals.
In the first one, we’ll have the FAQ expanded by default so it is present for the user to consume the data **OR** (as seen in the second screen) there will an information box after the selection since it is catchy enough to grab attention but not intrusive as to discourage users from contributing. 

Please keep in mind, we’ve now hidden the user’s email address after evaluating the comments under [this message.](https://opencollective.slack.com/archives/C0HSLRNVC/p1630656516043800)

| Expanded FAQ | Info Box | 
| --- | --- |
|![Expanded FAQ](https://user-images.githubusercontent.com/48468919/134652253-0e185302-6b40-4801-95db-d576d0a465ff.png) | ![Info Box](https://user-images.githubusercontent.com/48468919/134653210-d5f2ef15-c4b7-4f65-aa6a-a58ad5cdd89d.png)

**Comment by aminakazi** [comment]
[Production Ready file](https://www.figma.com/file/ZQBMWhnGGtRWeIZknFW1eP/%5BOC-Design%5D-Production-Ready-%E2%9C%85?node-id=14374%3A179238) is linked with the final designs.

**Comment by BenJam** [comment]
@Betree @znarf could we push this (and 'revert' https://github.com/opencollective/opencollective/issues/3017) monday morning?

**Comment by Betree** [comment]
@BenJam These are small changes, I can iterate today

**Comment by BenJam** [comment]
Looks like this can be closed out 🙌
